app/code/Magento/Theme/Block/Html/Topmenu.php

```diff
@@ -331,7 +331,7 @@ protected function _getMenuItemClasses(\Magento\Framework\Data\Tree\Node $item)
     /**
      * Add identity
      *
-     * @param array $identity
+     * @param string $identity
```

Comment:
technically this method accepts both strings and arrays as `in_array` will work with both and we never actually check the type. Since this class is noted with the `@api` I think it would be best to update the typehint to be `string|array` as we do not know how people use it.

Reply:
Thanks, never know that I can use `array` as identity.

Reply:
I mean technically you can, not sure you should do though.